packages/cli/src/daemon-config.ts

```diff
@@ -221,10 +221,13 @@ export class IndexingConfig {
   allowQueriesBeforeHistoricalSync = false
 
   /**
+   * @deprecated
```

Comment:
wasn't this already deprecated in another PR?  Does this PR need to update the base branch so we only see the new changes that are specific to this PR?

Reply:
good catch - just cross checked against `develop`. yeah i think that just came in because this branch was originally a fork of your WIP and I overlooked that deprecation during the merge from `develop`. I'll correct that in a minute. This should be the only file thats affected.